src/log.rs

```diff
@@ -24,15 +24,20 @@ byond_fn! { log_write(path, data) {
             Entry::Vacant(elem) => elem.insert(open(path)?),
         };
 
-        // write first line, timestamped
-        let mut iter = data.split('\n');
-        if let Some(line) = iter.next() {
-            write!(file, "[{}] {}\n", Utc::now().format("%F %T%.3f"), line)?;
-        }
+        let format_bool: bool = format.parse().unwrap();
```

Comment:
I don't know rust so take this with a keg of salt, but isn't it possible for this to throw?